reqs/management/commands/import_reqs.py

```diff
@@ -238,12 +239,22 @@ def fix_excel_decimals(self, reqid):
         Since the rows are sequential (or should be!), we should be able to
         account for this.
         """
+        while reqid.endswith("."):  # Handle a hack around Excel problems
+            reqid = reqid[:-1]
+        if "." not in reqid:
+            raise ValueError(
+                "Requirement ID without . separator: {0}".format(reqid))
         policy, requirement = reqid.split(".")
         last_policy, last_requirement = self.last_id.split(".")
         if policy == last_policy and last_requirement.endswith("9"):
             sequential = str(int(last_requirement) + 1)
             if sequential.rstrip("0") == requirement:
-                return "{0}.{1}".format(policy, sequential)
+                reqid = "{0}.{1}".format(policy, sequential)
```

Comment:
I think we still need it. If we have the following IDs:
1.08, 1.09, 1.1, 1.11
then the sequential logic will correctly change the third one to `1.10`. If we remove it, it'll get imported as `1.1`, which won't be a duplicate, but will need correcting later.